napari/resources/stylesheet.qss

```diff
@@ -433,8 +443,8 @@ QtControls {
     padding: 4px;
     min-height: 210px;
     max-height: 210px;
-    min-width: 220px;
-    max-width: 220px;
+    min-width: 240px;
+    max-width: 240px;
```

Comment:
one of the nice things about being able to float the controls would be the ability to gain a bit more horizontal accuracy on the range sliders... but the max-width limit here prevents you from resizing the window after floating.  What would you think about relaxing that?

Reply:
dropping that would be fine, but I can't figure out right now how to get the start-up width to be minimal (now 228), but still allow for it to be expanded. If you know can you suggest here / push the change to my PR